### DISCUSSION TITLE
RefinableValue filter in url

### DISCUSSION THREAD

**Discussion by Nevinia210** [microsoft-search/pnp-modern-search#3209] [Q&A] [answered]
Hello,
recently i updated pnp search webpart + search filters + results and in my case we used link in few pages to this search with some prefilled values that are basically described in url
previously we had url like: https://tenant.sharepoint.com/sites/news?filters=[{"n":"RefinableString03","t":["Logistics"]}]
now similiar url would look like: https://tenant.sharepoint.com/sites/news?f=[{"filterName":"RefinableString02","values":[{"name":"Company","value":"ǂǂ436f6d70616e79","operator":0}],"operator":"or"}]#

i would like to ask, where this "ǂǂ436f6d70616e79" comes from? what it is? how to find it?

**Comment by kasperbolarsen** [comment, marked as answer]
I just wrote about this a few days ago :-) See https://ms365thinking.blogspot.com/2023/08/pnp-modern-search-set-refiners-in-url.html

**Reply by Nevinia210** [reply]
perfect! thanks a lot :)

**Reply by Nevinia210** [reply]
@kasperbolarsen maybe one more thing to add there,
why there is a need to have this refiners in the url? in previous version there was no need to get them and they should use same searchquery, is it mandatory to have them? or can i get rid of them somehow, so they will not be displayed in url?
